### ISSUE TITLE
Feature request: option to not cycle color of new children

### ISSUE THREAD

**Issue by jeancf** [phase1geo/Minder#111]
Hi there,
I color code the children of a node to mean special things and I would like to have the option to have new children pick up the color of the last created child.

I.e. If I create child 1 with blue link color, when I create child 2 I want it to be blue by default. the same for 3, etc. until I manually change the color of the child to red. Then the next child will be created in red by default.

P.S. The more I use this tool the more I love it!

**Comment by phase1geo** [comment]
This behavior would certainly be different from the default behavior.  Do you have a suggestion for how Minder should use this type of behavior instead of the current behavior (i.e., I would like to preserve the current behavior)?  Preference option? Something else?

**Comment by phase1geo** [comment]
For context, other than children of the root node of a tree; all other nodes inherit the color of their parent node.

**Comment by jeancf** [comment]
>  Do you have a suggestion for how Minder should use this type of behavior

I was thinking of just having a check box with that would enable/disable the default color rotation behavior. It could be called "Vary color on create" or something. It is checked by default and when unchecked, the color of the last created child is used for any new node.

> For context, other than children of the root node of a tree; all other nodes inherit the color of their parent node.

I understand that and that is great. It is the level just below the root node that I am interested in.µ

**Comment by phase1geo** [comment]
I have updated the development branch to include support for this feature request which will be generally available in the next stable release.

I have added a new switch in the map sidebar panel which controls the rotation of the link color selection.  It should be set be default which mimics the normal behavior of the application; however, if the switch is cleared, it will cause the last used link color to be used when the next main branch node is created.
